Source/WebInspectorUI/UserInterface/Views/DOMTreeContentView.js

```diff
@@ -529,6 +529,14 @@ WI.DOMTreeContentView = class DOMTreeContentView extends WI.ContentView
                 WI.settings.enabledDOMTreeBadgeTypes.save();
             }, WI.settings.enabledDOMTreeBadgeTypes.value.includes(WI.DOMTreeElement.BadgeType.Event));
         }
+
+        // COMPATIBILITY (macOS 13.0, iOS 16.0): `Scrollable` value for `CSS.LayoutFlag` did not exist yet.
+        if (InspectorBackend.Enum.CSS.LayoutFlag.Scrollable) {
```

Comment:
This should probably be
```
InspectorBackend.Enum.CSS?.LayoutFlag?.Scrollable
```
just in case the target doesn't have a `CSS` domain or it's too old for `CSS.LayoutFlag` to exist.